### ISSUE TITLE
"MySQL restore" database window

### ISSUE THREAD

**Issue by sh3bang** [dbeaver/dbeaver#34655]
I miss a hint in the "dump restore" window on which connection the import will be made. This would be particularly helpful and more secure if you have configured multiple connections! 

DBeaver have a helpful function to color different connections in different colors (e.g. Production = Red, local development = green, ... and so on). But the restore window dont use colors and dont have any text information:

![dbeaver-restore](https://github.com/dbeaver/dbeaver/assets/6939568/6508850c-6914-43f7-8fc8-d9c79d03d896)

**Comment by E1izabeth** [comment]
I agree with you. There is no clue about database connection at all.

**Comment by E1izabeth** [comment]
Thank you for request
